django-stubs/dispatch/dispatcher.pyi

```diff
@@ -19,7 +25,14 @@ class Signal:
     ) -> bool: ...
     def has_listeners(self, sender: Any = ...) -> bool: ...
     def send(self, sender: Any, **named: Any) -> list[tuple[Callable, str | None]]: ...
-    def send_robust(self, sender: Any, **named: Any) -> list[tuple[Callable, Exception | str]]: ...
+    def send_robust(self, sender: Any, **named: Any) -> list[tuple[Callable, Exception | Any]]: ...
     def _live_receivers(self, sender: Any) -> list[Callable]: ...
 
-def receiver(signal: list[Signal] | Signal, **kwargs: Any) -> Callable: ...
+def receiver(
+    signal: list[Signal] | tuple[Signal, ...] | Signal,
+    *,
+    receiver: Callable,
```

Comment:
Can we annotate arg types? Maybe using `ParamSpec`?

Reply:
Good idea! I also removed the receiver argument, which should not have been there.